### PR TITLE
Added Tiltfile for running opencrvs on kuberntes locally

### PR DESCRIPTION
**Pull request by adskyiproger**
## Description

This PR aims to introduce kubernetes support for OpenCRVS Core services.

## Checklist

- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [x] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly: N/A

Tilt dashboard:
image

Data seed:
image

Integration with minio works fine:
image

### CODE REVIEW

File: Tiltfile [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        "countryconfig.image.tag={}".format(countryconfig_image_tag)
+        ]
+      )
+)
+
+######################################################
+# Data management tasks:
+# - Reset database: This task is not part of helm deployment to avoid accidental data loss
+# - Seed data: is part of helm install post-deploy hook, but it is a manual task as well
+# - Run migration job, is part of helm install/upgrade post-deploy hook
```
Comment (cibelius) [inline Tiltfile:144]: Nice comments all the way 💪 🤩

**Review by cibelius — APPROVED** [review]
So far looks like this works on my machine as well! (still downloading images)

Nice and easy setup 👍 

Screenshot 2025-04-09 at 15 48 23
